src/SFA.DAS.EmployerCommitmentsV2.Web/Authentication/IUserService.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace SFA.DAS.EmployerCommitmentsV2.Web.Authentication
+{
+    public interface IUserService
```

Comment:
What do you think to renaming this to `IAuthenticationService` to be consistent with `SFA.DAS.ProviderCommitments`?

Reply:
I actually think calling it a 'UserService' is more correct as it's not an authentication service, as the user is not necessarily authenticated when it gets called

Reply:
Yeah it's a tough one to name. It's called `IAuthenticationService` in a few projects now though so I think consistency should win here.